.github/workflows/resolve-register-conflict.yml

```diff
@@ -0,0 +1,121 @@
+          PR_LIST=$(gh pr list --json headRefName,labels --search "org:okp4 repo:okp4/networks state:open label:register-validator author:bot-anik sort:created-asc")
+          NUM_OF_PR=$(echo $PR_LIST | jq '. | length')
+          if [[ $NUM_OF_PR = 0 ]]; then
+            echo No PR waiting
+            exit
+          fi
+          OLDEST_PR=$(echo $PR_LIST | jq '.[0]')
+          CHAIN_LABEL=$(echo $OLDEST_PR | jq .labels | jq -r ".[] | select(.name | startswith(\"${LABEL_PREFIX}\"))" | jq .name)
+          CHAIN_ID=$(echo ${CHAIN_LABEL/${LABEL_PREFIX}/} | tr -d ' ')
+          if [ -z "$CHAIN_ID" ]; then
+            echo "No chain id on label"
+            exit
+          fi
+
+          echo "chain-id=${CHAIN_ID}" >> $GITHUB_OUTPUT
+          echo "headRefName=$(echo $OLDEST_PR | jq .headRefName )" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.OKP4_TOKEN }}
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v5
+        with:
+          gpg_private_key: ${{ secrets.OKP4_BOT_GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.OKP4_BOT_GPG_PASSPHRASE }}
+          git_config_global: true
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+
+      - name: rebase and resolve genesis conflicts
+        id: resolve-conflict
+        run: |
+          set -eo pipefail
+          CHAIN_ID=${{ steps.next-pr.outputs.chain-id }}
+
+          git switch ${{ steps.next-pr.outputs.headRefName }}
+
+          if [ -z "$(git diff --check  main)" ]; then
+            echo No merge conflicts
+            echo "conflict-exists=false" >> $GITHUB_OUTPUT
+            exit 0
+          else
+            echo "conflict-exists=true" >> $GITHUB_OUTPUT
+          fi
+
+          git diff @~ @ -- chains/$CHAIN_ID/genesis.json > temp-diff.patch
+          PATCH=$(grep ^\+ temp-diff.patch | cut -c2-)
+          ACCOUNT_ELT=$(echo $PATCH | grep -o -E '(\{ "@type": "/cosmos.auth.v1beta1.BaseAccount"[^}]*})')
+          BALANCE_ELT=$(echo $PATCH | grep -o -E '(\{ "address": [^}]*} ] })')
+          GENTX_ELT=$(echo $PATCH | grep -o -E '(\{ "body".*)')
+          if [[ $GENTX_ELT != *} ]]; then
+            GENTX_ELT=$(echo $GENTX_ELT"}")
+          fi
+          if [ "$(echo $ACCOUNT_ELT | jq .address)" = 'null' ]; then
+            echo wrong account elt >&2
+            exit 1
+          fi
+          if [ "$(echo $BALANCE_ELT | jq .address)" = 'null' ]; then
+            echo wrong balance elt >&2
+            exit 1
+          fi
+          if [ "$(echo $GENTX_ELT | jq .body.messages[0].description.moniker)" = 'null' ]; then
+            echo wrong gentx elt >&2
+            exit 1
+          fi
+
+          git show main:chains/$CHAIN_ID/genesis.json \
+          | jq ".app_state.auth.accounts += [$ACCOUNT_ELT]" \
+          | jq ".app_state.bank.balances += [$BALANCE_ELT]" \
+          | jq ".app_state.genutil.gen_txs += [$GENTX_ELT]" \
+            > genesis-temp.json
```

Comment:
Extracting data from the git diff and re-applying them with jq seems dangerous to me because for too reasons:
- jq will format in a certain way and okp4d may format differently, 
- we depend on the diff identified which is hard to predict

Why not simply use okp4d and run the `add-genesis-account` & `collect-gentxs` commands?

Reply:
oh 🙄 yes way better to process the genesis from main add re apply the gentx, I think I'll do an other PR...

Reply:
testing in progress 👷...